ui/apps/platform/src/Containers/SystemHealth/UsageStatistics/UsageStatisticsForm.tsx

```diff
@@ -0,0 +1,168 @@
+            .then((usage) => {
+                setMaxUsage(usage);
+                setErrorFetchingMax('');
+            })
+            .catch((error) => {
+                setErrorFetchingMax(getAxiosErrorMessage(error));
+            });
+    }, [startDate, endDate]);
+
+    return (
+        <>
+            {errorFetchingCurrent && (
+                <Alert isInline title={errorFetchingCurrent} variant="danger" />
+            )}
+            {errorFetchingMax && <Alert isInline title={errorFetchingMax} variant="danger" />}
+            <div className="pf-u-font-size-lg">Currently secured</div>
+            <Divider style={{ padding: '5px 0px 10px 0px' }} />
+            <DescriptionList
+                columnModifier={{
+                    default: '2Col',
+                }}
+            >
+                <DescriptionListGroup>
+                    <DescriptionListTerm>CPU units</DescriptionListTerm>
+                    <DescriptionListDescription>
+                        {currentUsage.numCpuUnits}
+                    </DescriptionListDescription>
+                </DescriptionListGroup>
+                <DescriptionListGroup>
+                    <DescriptionListTerm>Nodes count</DescriptionListTerm>
+                    <DescriptionListDescription>{currentUsage.numNodes}</DescriptionListDescription>
+                </DescriptionListGroup>
+            </DescriptionList>
+
+            <div className="pf-u-font-size-lg" style={{ padding: '20px 0px 0px 0px' }}>
+                Maximum secured
+            </div>
+            <Divider style={{ padding: '5px 0px 10px 0px' }} />
+            <DescriptionList
+                columnModifier={{
+                    default: '2Col',
+                }}
+            >
+                <DescriptionListGroup>
+                    <DescriptionListTerm>Start date</DescriptionListTerm>
+                    <DescriptionListDescription>
+                        <DatePicker
+                            value={yyyyMMddFormat(startDate)}
+                            onChange={(_str, _, date) => {
+                                if (date) {
+                                    setStartDate(date);
+                                }
+                            }}
+                        />
+                    </DescriptionListDescription>
+                </DescriptionListGroup>
+                <DescriptionListGroup>
+                    <DescriptionListTerm>End date</DescriptionListTerm>
+                    <DescriptionListDescription>
+                        <DatePicker
+                            value={yyyyMMddFormat(endDate)}
+                            onChange={(_str, _, date) => {
+                                if (date) {
+                                    // Add 23 hours 59 minutes to include end date completely in the request.
+                                    date.setTime(date.getTime() + (24 * 60 - 1) * 60 * 1000);
```

Comment:
```suggestion
                                    // Add 1 day to include end date completely in the request.
                                    date.setUTCDate(date.getUTCDate() + 1);
```

Reply:
If we do that, date will change in the form as well

Reply:
Oh, then, we should only do that when querying the API.